src/FixedDataTableRoot.js

```diff
@@ -13,15 +13,17 @@
 
 import Table from 'FixedDataTableContainer';
 import Cell from 'FixedDataTableCellDefault';
+import CellDeprecated from 'FixedDataTableCellDefaultDeprecated';
 import Column from 'FixedDataTableColumn';
 import ColumnGroup from 'FixedDataTableColumnGroup';
 
 const version = '1.1.0';
 
 export {
   Cell,
+  CellDeprecated,
```

Comment:
Users who still wanna use the old cell will need to export `CellDeprecated`

Reply:
This would still be a breaking change in terms of API right.  How about naming the new cell something like DataCell to avoid name conflict? CC: @wcjordan

Reply:
I'm all for not breaking the API until a major update.  DataCell sounds fine to me.